source/appendices/e-audit-requirements-and-tests/03-pr-tests.rst

```diff
@@ -0,0 +1,85 @@
+.. The COUNTER Code of Practice Release 5 © 2017-2021 by COUNTER
+   is licensed under CC BY-SA 4.0. To view a copy of this license,
+   visit https://creativecommons.org/licenses/by-sa/4.0/
+
+E.3 Platform Report Audit Tests
+-------------------------------
+
+This section of the appendix outlines tests that MUST be run during all audits, and which are specific to the Platform Report.
+
+The Platform Report will be COUNTER-compliant if the following audit tests are passed. The Platform Usage (PR_P1) Standard View of the Platform Report will be COUNTER-compliant if the metrics match those in the Platform Report, with the appropriate aggregation.
+
+Audit test requirements vary depending on the set up of the platform, as indicated by the Options within the tests below.
+
+Platform Report tests apply to all Host_Types.
+
+
+E.3.1 Searches_Platform
+"""""""""""""""""""""""""
+
+**Option 1**: Platform has multiple databases and the user can search the whole platform, multiple selected databases, or a single database.
+
+The auditor MUST run 100 searches on the platform
+
+* 50 searches against only 1 selected database.
+* 25 searches against 2 selected databases.
+* 25 searches against all databases on the platform.
+
+The auditor activity MUST result in 100 Searches_Platform (1 per search).
+
+**Option 2**: Platform has multiple databases and the user can search the whole platform or a single database.
+
+The auditor MUST run 100 searches on the platform
+
+* 50 searches against only 1 selected database.
+* 50 searches against all databases on the platform.
+
+The auditor activity MUST result in 100 Searches_Platform (1 per search).
+
+**Option 3**: Platform has multiple databases but the user can only search the whole platform OR the platform has just one database.
+
+The auditor MUST run 100 searches on the platform.
+
+The auditor activity MUST result in 100 Searches_Platform (1 per search).
+
+**Option 4**: Platform has multiple databases but the user can only search a single database at a time.
+
+The auditor MUST run 100 searches across at least 10% of the available databases, restricted to one database at a time.
+
+The auditor activity MUST result in 100 Searches_Platform (1 per search).
+
+**Option 5**: Platform does not include any databases.
+
+The auditor MUST run 100 searches on the platform.
+
+The auditor activity MUST result in 100 Searches_Platform (1 per search).
+
+E.3.2 Platform Investigations and Requests
```

Comment:
Option 1 and 2 don't cover the case that only whole book downloads are offered and segments are identifiable.